Source/Common/TimeSpanToISO8601DurationValuesJsonConverter.cs

```diff
@@ -30,18 +31,18 @@ public override void WriteObject(JsonWriter writer, object? value, JsonSerialize
                 throw new ArgumentNullException(nameof(value));
```

Comment:
Should we be throwing here? `value` is nullable.

Reply:
That's a good point - this is wrong.